modules/cnv_master/1.0/config/default.yaml

```diff
@@ -0,0 +1,45 @@
+lcr-modules:
+
+    cnv_master:
+        # comma-separated list of modules used to call CNV
+        # this order will also determine the preference which module's output will be used when multiple outputs are available
+        # example: ["battenberg", "sequenza", "cnvkit", "controlfreec"]
+        names: ["__UPDATE__", "__UPDATE__", ..., "__UPDATE__"]
+        # comma-separated list of path to the location of seg files
+        # must be in the same order as the names of the tools in previous key
+        # supported wildcards are seq_type, tumour_id, normal_id, pair_status, tool, projection
+        # example: ['results/battenberg-1.2/99-outputs/seg/{seq_type}--projection/{tumour_id}--{normal_id}--{pair_status}.{tool}.{projection}.seg',
+        #    'results/sequenza-1.4/99-outputs/seg/{seq_type}--projection/{tumour_id}--{normal_id}--{pair_status}.{tool}.{projection}.seg',
+        #    'results/cnvkit-1.0/99-outputs/seg/{seq_type}--projection/{tumour_id}--{normal_id}--{pair_status}.{tool}.{projection}.seg',
+        #    'results/controlfreec-1.2/99-outputs/seg/{seq_type}--projection/{tumour_id}--{normal_id}--{pair_status}.{tool}.{projection}.seg'
+        #    ]
+        seg: ["__UPDATE__",
+            "__UPDATE__",
+            ...,
+            "__UPDATE__"
+            ]
+        # requested projections of the merged files
+        projections: ["grch37", "hg38"]
+
+        scratch_subdirectories: []
+
+        conda_envs:
+            R: "{MODSDIR}/envs/readr-2.1.2.yaml"
+
+        threads:
+            cnv_master: 4
+
+        resources:
+            cnv_master:
+                mem_mb: 12000
+                disk_mb: 1000 # Will ensure at least this much space is left
+
+        pairing_config:
```

Comment:
Will this cause compatibility issues with Battenberg since it's not run with unmatched normals? Or is that handled by the input function that checks for availability of the output file?

Reply:
Right, it is handled by the input function. This pairing config here will only ensure that the output of any tool is present for any sample - so all tumors in the sample table are included in the merge.